legend-engine-xt-authentication-implementation-core/pom.xml

```diff
@@ -128,13 +128,13 @@
         <dependency>
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcprov-jdk15on</artifactId>
-            <version>1.70</version>
+            <version>1.67</version>
```

Comment:
legend-pure uses 1.67. It was causing a lot of dependency issues. Ephrim added it for some snowflke tests. This ia part of this PR : https://github.com/finos/legend-engine/pull/1417
It will go once i rebase.